Update Ruby README's w/ WRITEME ✍️ (#4758)
awsdocs/aws-doc-sdk-examples: ruby/example_code/sns/README.md

* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).

<!--custom.important.start-->
<!--custom.important.end-->

## Code examples
### Single actions

Code excerpts that show you how to call individual service functions.

* [Create a topic](sns-ruby-example-create-topic.rb#L12) (`CreateTopic`)
* [List the subscribers of a topic](sns-ruby-example-show-subscriptions.rb#L12) (`ListSubscriptions`)
* [List topics](sns-ruby-example-show-topics.rb#L11) (`ListTopics`)
* [Publish to a topic](sns-ruby-example-send-message.rb#L13) (`Publish`)
* [Set topic attributes](sns-ruby-example-enable-resource.rb#L14) (`SetTopicAttributes`)
* [Subscribe an email address to a topic](sns-ruby-example-create-subscription.rb#L13) (`Subscribe`)

## Run the examples

### Prerequisites

For prerequisites, see the [README](../../README.md#Prerequisites) in the `ruby` folder.

<!--custom.prerequisites.start-->
<!--custom.prerequisites.end-->

### Instructions
<!--custom.instructions.start-->

> The easiest way to interact with this example code is by invoking a [Scenario](#Scenarios) from your command line. For example, `ruby some_scenario.rb` will invoke `some_scenario.rb`.

Reviewer: ```suggestion
The quickest way to interact with this example code is to invoke a [Scenario](#Scenarios) from your command line. For example, `ruby some_scenario.rb` will invoke `some_scenario.rb`.
```

<!--custom.instructions.end-->

### Tests
<!--custom.tests.start-->
The example code in this directory is not currently tested.

## Contribute

> Code examples thrive on community contribution!

Reviewer: ```suggestion
Code examples thrive on community contribution.
```

> To learn more about the contributing process, see [CONTRIBUTING.md](../../../CONTRIBUTING.md)

Reviewer: ```suggestion
To learn more about the contributing process, see [CONTRIBUTING.md](../../../CONTRIBUTING.md).
```
